alexandria/core/utils.py

```diff
@@ -0,0 +1,41 @@
+from django.conf import settings
+from django.core.exceptions import ValidationError
+
+from alexandria.core import models
+
+
+def create_document_file(user, group, document_attributes, file_attributes):
```

Comment:
Please add a docstring, as these functions will likely be "public API"

Reply:
Add reST docs or was the idea something different